packages/client/src/views/Performance/Leaderboards.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+
+import React from 'react'
+import { Icon } from '@opencrvs/components/lib/Icon'
+import { DashboardEmbedView } from '@client/views/Performance/Dashboard'
+import { useIntl } from 'react-intl'
+import { messages } from '@client/i18n/messages/views/dashboard'
+
+export const Leaderboards = () => {
+  const intl = useIntl()
+  return (
+    <DashboardEmbedView
+      title={intl.formatMessage(messages.leaderboardTitle)}
+      url={window.config.LEADERBOARDS_DASHBOARD_URL}
```

Comment:
@rikukissa do we not need a PR in Farajaland where these are set in window.config client-config.js etc?

Reply:
Sorry, completely forgot to add it:
https://github.com/opencrvs/opencrvs-farajaland/pull/501